salt/modules/junos.py

```diff
@@ -1210,30 +1210,34 @@ def install_config(path=None, **kwargs):
                 elif not check:
                     try:
                         cu.rollback()
-                        ret[
-                            "message"
-                        ] = "Loaded configuration but commit check failed, hence rolling back configuration."
+                        ret["message"] = (
+                            "Loaded configuration but commit check failed, hence"
+                            " rolling back configuration."
+                        )
                     except Exception as exception:  # pylint: disable=broad-except
-                        ret[
-                            "message"
-                        ] = 'Loaded configuration but commit check failed, and exception occurred during rolling back configuration "{}"'.format(
-                            exception
+                        ret["message"] = (
+                            "Loaded configuration but commit check failed, and"
+                            ' exception occurred during rolling back configuration "{}"'.format(
+                                exception
+                            )
                         )
                         _restart_connection()
 
                     ret["out"] = False
                 else:
                     try:
                         cu.rollback()
-                        ret[
-                            "message"
-                        ] = "Commit check passed, but skipping commit for dry-run and rolling back configuration."
+                        ret["message"] = (
+                            "Commit check passed, but skipping commit for dry-run and"
+                            " rolling back configuration."
+                        )
                         ret["out"] = True
                     except Exception as exception:  # pylint: disable=broad-except
-                        ret[
-                            "message"
-                        ] = 'Commit check passed, but skipping commit for dry-run andi while rolling back configuration exception occurred "{}"'.format(
-                            exception
+                        ret["message"] = (
+                            "Commit check passed, but skipping commit for dry-run andi"
```

Comment:
Not a blocker but a minor typo we could fix while making changes here.

Reply:
Fixed